addons/bus/static/src/outdated_page_watcher_service.js

```diff
@@ -47,7 +47,7 @@ export class OutdatedPageWatcherService {
     }
 
     async checkHasMissedNotifications() {
-        if (!this.multi_tab.isOnMainTab()) {
+        if (!this.multi_tab.isOnMainTab() || !this.lastNotificationId) {
```

Comment:
IMO, we can't fix it here. Because this issue was generated due to a network issue, we don't have the value of `last_notification_id` in `getSharedValue`.

Instead, you should fix the below as `multi_tab.getSharedValue("last_notification_id", 0)` so will get notification for outdated page 
https://github.com/odoo/odoo/blob/d8b4bf6a6e3eceacb356677043705354c7dc8593/addons/bus/static/src/outdated_page_watcher_service.js#L19
https://github.com/odoo/odoo/blob/d8b4bf6a6e3eceacb356677043705354c7dc8593/addons/bus/static/src/outdated_page_watcher_service.js#L32